ui/src/components/review/ReviewTable.js

```diff
@@ -165,6 +164,10 @@ class ReviewTable extends React.Component {
                     this.props.onUpdateSuccess(
                         `Successfully submitted the review for role ${this.props.role}`
                     );
+                    this.setState({
```

Comment:
does react componentDidUpdate method work with redux? 
Instead of setting the state again here we can use componentDidUpdate and if props has changed then update the state